mockwebserver/src/main/kotlin/mockwebserver3/MockWebServer.kt

```diff
@@ -409,12 +408,12 @@ public class MockWebServer : Closeable {
     val serverSocket = this.serverSocket ?: return // If this is null, start() must have failed.
 
     // Cause acceptConnections() to break out.
-    serverSocket.close()
+    serverSocket.closeQuietly()
 
     // Await shutdown.
     for (queue in taskRunner.activeQueues()) {
       if (!queue.idleLatch().await(5, TimeUnit.SECONDS)) {
-        throw IOException("Gave up waiting for queue to shut down")
+        throw AssertionError("Gave up waiting for queue to shut down")
```

Comment:
If this occurs in practice . . . the system is unhealthy. I think crashing hard like this is okay.